docs/authentication/oidc.md

```diff
@@ -97,3 +97,35 @@ AUTH_USER_METHOD = get_auth_user
 ```
 
 Once done, you'll have the end-to-end authentication in Amundsen without any proxy or code changes.
+
+
+## Using Okta with Amundsen on K8s
+Assumptions:
+- You have access to okta (you can create a developer account for free!)
+- You are using k8s to setup amundsen. See [amundsen-kube-helm](../../amundsen-kube-helm/README.md)
+
+1. You need to have a stable DNS entry for amundsen-frontend that can be registered in okta.
+    - for example in AWS you can setup route53
+    I will assume for the rest of this tutorial that your stable uri is "http://amundsen-frontend"
+2. You need to register amundsen in okta as an app. More info [here](https://developer.okta.com/blog/2018/07/12/flask-tutorial-simple-user-registration-and-login). 
+But here are specific instructions for amundsen:
+    - At this time, I have only succesfully tested integration after ALL grants were checked.
+    - Set the Login redirect URIs to: http://amundsen-frontend/oidc_callback
+    - No need to set a logout redirect URI
+    - Set the Initiate login URI to: http://amundsen-frontend/oidc_callback
+    - Copy the Client ID and Client secret as you will need this later.
+3. When you start up helm you will need to provide some properties. Here are the properties that need to be overridden for oidc to work:
+    ```yaml
+    oidcEnabled: true
+    createOidcSecret: true
+    OIDC_CLIENT_ID: YOUR_CLIENT_ID
+    OIDC_CLIENT_SECRET: YOUR_SECRET_ID
+    OIDC_ORG_URL: https://edmunds.okta.com
+    OIDC_AUTH_SERVER_ID: default
+    ```
+4. At present, there is no oidc build of the frontend. So you will need to build an oidc build yourself and upload it to, for example ECR, for use by k8s.
```

Comment:
if it would be useful, I can provide more info on how to build frontend with oidc

Reply:
thanks. I think it would be helpful.